vendor-extra/JatsContentBundle/tests/ViewConverter/ArticleTitleHeadingVisitorTest.php

```diff
@@ -85,17 +88,32 @@ public function it_does_nothing_if_there_is_already_text_set() : void
      */
     public function it_sets_the_text_argument() : void
     {
-        $visitor = new ArticleTitleHeadingVisitor();
-
-        $xml = FluentDOM::load('<article-title xmlns="http://jats.nlm.nih.gov">foo <bar>baz</bar></article-title>');
+        $visitor = new ArticleTitleHeadingVisitor($this->createInlineConverter());
+
+        $xml = FluentDOM::load(
+            <<<XML
+<jats:article-title xmlns:jats="http://jats.nlm.nih.gov">
+    foo <jats:italic>bar</jats:italic> baz
+</jats:article-title>
+XML
+        );
         /** @var Element $element */
         $element = $xml->documentElement;
 
         $newContext = ['foo' => 'bar'];
         $view = $visitor->visit($element, new View('@LiberoPatterns/heading.html.twig'), $newContext);
 
         $this->assertSame('@LiberoPatterns/heading.html.twig', $view->getTemplate());
-        $this->assertEquals(['text' => 'foo baz'], $view->getArguments());
+        $this->assertEquals(
+            [
+                'text' => [
+                    new View(null, ['object' => '/jats:article-title/text()[1]', 'context' => ['foo' => 'bar']]),
```

Comment:
too much `foo bar` doesn't explain where things go; I am mapping this to 
```
foo <jats:italic>bar</jats:italic>
```
which means... the text before italic becomes the key of a `context` array value, which is the content of the italic tag? :thinking:

Reply:
Heh, will change the context names.

Reply:
Incidentally, if the various `foo` are not meant to be the same thing, mutation testing may find it as a test not failing when variables are mixed up (but it's a brute force approach, and other tests may fail anyway)